lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -58,7 +58,7 @@ def load_graph(url:)
         def normalize_results
           normalize_start_dt = Time.now.utc
 
-          @graph = graph_service.filter(graph: @graph, language: language, remove_blanknode_subjects: true)
+          @filtered_graph = graph_service.filter(graph: @full_graph, language: language)
           results = map_results
```

Comment:
This filtering is unnecessary.  [SearchResultsMapperService](https://github.com/samvera/questioning_authority/blob/master/app/services/qa/linked_data/mapper/search_results_mapper_service.rb#L68) ignores blank nodes when looking for potential subject uri results.

Reply:
There are no test changes because removing here and ignoring in SearchResultsMapperService was redundant.